services/PostService.js

```diff
@@ -10,7 +10,7 @@ class PostService {
   }
 
   static createNewPost(title, author, body) {
-    return new Post({ title, author, body }).save();
+    return Post.create({ title, author, body });
```

Comment:
This is good; it's common that `new` makes things harder to test

Reply:
Nice. I didn't know how to use Sinon's stub format of sinon.stub(object, method of object); if it was Post().save();